### DISCUSSION TITLE
New destination: Exasol

### DISCUSSION THREAD

**Discussion by alafanechere** [airbytehq/airbyte#33156] [New Connector Request]
## Tell us about the new connector you’d like to have
[Exasol](https://www.exasol.com/) as advertised on our [website](https://airbyte.com/connectors/exasol)

## Describe the context around this new connector
Requested in this [Slack conversation](https://airbytehq.slack.com/archives/C019WJFR1JT/p1641983280065700)

**Comment by itaseskii** [comment]
Hey @alafanechere has anyone starting working on this? Is it up for taking?

**Comment by alafanechere** [comment]
Sure! Feel free to jump on this.

**Comment by ThomasBestfleisch** [comment]
I'm currently working on the destination and once the acceptance tests are green I will submit a PR

**Comment by grishick** [comment]
@ThomasBestfleisch how is this coming along?

**Comment by itaseskii** [comment]
@grishick There is already a [PR](https://github.com/airbytehq/airbyte/pull/21200) for an Exasol destination connector that is already reviewed/approved and ready to be published/merged. My rights have been revoked so I can no longer do that, someone from Airbyte will need to take over.

**Comment by grishick** [comment]
I'll take it over

[a comment by an automated account is not shown]
